jbpm-services/jbpm-kie-services/src/test/resources/multi-remotable/pojo-pom.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+ <modelVersion>4.0.0</modelVersion>
+
+ <groupId>org.test</groupId>
+ <artifactId>jbpm-kie-services-filter-test-pojo</artifactId>
+ <version>1.0.0-SNAPSHOT</version>
+
+ <dependencies>
+  <dependency>
+   <groupId>org.kie</groupId>
+   <artifactId>kie-api</artifactId>
+   <version>6.4.0.Final</version>
```

Comment:
I believe we should use `${version.org.kie}` and setup maven-resource-plugin to filter this file (and thus replace the placeholder with actual version). If we hardcode `6.5.1-SNAPSHOT` it won't get automatically replaced.